org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -112,10 +112,10 @@ GithubOidcSageBionetworksSchematicInfra:
       - "arn:aws:iam::aws:policy/AdministratorAccess"
       - "arn:aws:iam::aws:policy/AWSKeyManagementServicePowerUser"
   TemplatingContext:
-    GitHubOrg: "Sage-Bionetworks"
+    GitHubOrg: "Sage-Bionetworks-IT"
```

Comment:
I think it would be better to add support for schematic-infra-v2 in one PR, and remove schematic-infra in another PR after the migration just in case you need to run another deployment for schematic-infra during the migration

Reply:
done